src/App.tsx

```diff
@@ -1,61 +1,155 @@
+import { FormEvent, useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { TodoList } from './components/TodoList';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+function getUser(todoUserId: number): User | null {
+  const userObj = usersFromServer.find(user => user.id === todoUserId);
+
+  return userObj || null;
+}
+
+export const newTodos: Todo[] = todosFromServer.map((todo) => {
+  return {
+    id: todo.id,
+    title: todo.title,
+    userId: todo.userId,
+    completed: todo.completed,
+    user: getUser(todo.userId),
+  };
+});
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [userName, setOption] = useState('0');
+  const [tittleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
```

Comment:
If you store a `boolean` type - start name with `is`, `has`
```suggestion
  const [isUserError, setIsUserError] = useState(false);
```